### PR TITLE
Simple static location sharing

### PR DESCRIPTION
**Pull request by ara4n**
Very basic and un-designed location sharing, intended as a quick & dirty prototype for future embellishment. **Not to be unlabsed until it's actually gone through the design team**.

Screenshot 2021-11-22 at 21 00 40

Twinned with https://github.com/matrix-org/matrix-react-sdk/pull/7135

---
Here's what your changelog entry will look like:

## ✨ Features
 * Simple static location sharing ([\#19754](https://github.com/vector-im/element-web/pull/19754)).

### CODE REVIEW

**Comment by ShadowJonathan** [comment]
(Simple note, but a screenshot/design reference would probably help here)

**Review by gsouquet — APPROVED** [review]
Looks pretty sensible to me! ✨

File: config.sample.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "features": { },
     "default_federate": true,
     "default_theme": "light",
+    "map_style_url": "https://api.maptiler.com/maps/basic/style.json?key=YOUR_KEY_GOES_HERE",
```
Comment (t3chguy) [inline config.sample.json:31]: including this in the sample config will just cause most deployments to end up with this placeholder API Key which will lead to many unexpected experiences, imo this shouldn't be in the sample config

**Review by t3chguy — APPROVED** [review]
lgtm otherwise

**Review by t3chguy — APPROVED** [review]

**Pull request merged**